bentoml/_internal/io_descriptors/image.py

```diff
@@ -1,11 +1,11 @@
 import io
+import mimetypes
 import typing as t
 
+import imageio
```

Comment:
`imageio` is no longer a dependency, is it possible to remove it? ideally we want to use only `pillow`, which is imported with lazyloader below. cc @aarnphm @bojiang

Reply:
this is breaking the documentation checker rn

Reply:
I will update it to Pillow

Reply:
oh never expected that. yeah these commits is for quick testing, not well formated for dependencies